CHANGELOG.md

```diff
@@ -113,8 +134,6 @@
 
 - Emergency patch for [PFM](https://github.com/cosmos/ibc-apps/tree/main/middleware).
   ([\#2980](https://github.com/cosmos/gaia/pull/2980))
-- Bump [PFM](https://github.com/cosmos/ibc-apps/tree/main/middleware) to `v7.1.3-0.20240228213828-cce7f56d000b`.
```

Comment:
are we deleting this here on purpose? seems like we should have it in both places, right?

Reply:
That was a mistake as Gaia v14.2.0 clearly didn't update PFM to v7 as that requires SDK 0.47.

Reply:
Oh, okay, yeah makes sense.